src/main/java/Storage.java

```diff
@@ -0,0 +1,125 @@
+import Task.Deadline;
+import Task.Event;
+import Task.Task;
+import Task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Returns File object from a string.
+     *
+     * @param filePath location of the file as a string.
+     * @return File object.
+     */
+    public File getFile(String filePath) {
+        File f = new File(filePath);
+        return f;
+    }
+
+    /**
+     * Updates the .txt with the most recent changes.
+     *
+     * @param filePath location of the file as a string.
+     * @param list The existing ArrayList of Tasks after user has made changes to it.
+     */
+    public void updateTasks(String filePath, ArrayList<Task> list) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        String result = "";
+        for (int i = 0; i < list.size(); i++) {
+            Task task = list.get(i);
+            String c = task.getClass().toString();
+            if (c.equals("class Task.Todo")) {
```

Comment:
I recommend using `instanceof` keyword to check class type. It is more reliable and easier to implement than string comparison. [Reference](https://www.javatpoint.com/downcasting-with-instanceof-operator)

Reply:
Good idea! Thanks